scene/gui/color_picker.cpp

```diff
@@ -2177,7 +2179,7 @@ void ColorPicker::_bind_methods() {
 
 	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_normal, "tab_unselected", "TabContainer");
 	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_pressed, "tab_selected", "TabContainer");
-	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_hover, "tab_selected", "TabContainer");
+	BIND_THEME_ITEM_EXT(Theme::DATA_TYPE_STYLEBOX, ColorPicker, mode_button_hover, "tab_hovered", "TabContainer");
```

Comment:
Not important, but this style has visible borders at sides:
![image](https://github.com/user-attachments/assets/c5b35f49-6e85-4ef2-b74b-162548a63801)

Reply:
Also the `tab_unselected` has the same left and right borders but it's less visible, was thinking about removing those borders but less visible.

They are clear here.
![image](https://github.com/user-attachments/assets/c76fd6e6-8843-4f9e-acc4-46d599f19eb7)